tests/libcxx-backports/mdspan/mdspan.index_operator.pass.cpp

```diff
@@ -38,6 +39,9 @@
 #include "../ConvertibleToIntegral.h"
 #include "../CustomTestLayouts.h"
 
+// This test uses the bracket operator, but its not something we have at configure time
+#if MDSPAN_USE_BRACKET_OPERATOR
```

Comment:
I think we should still test these with the operator() equivalents otherwise we may have holes in our test (e.g. C++20 span call but you mentioned above)

Reply:
No I want to keep the libcxx-backports as close to the LLVM files as possible. I.e. the diffs should be minimal.